### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by vitoyucepi**
## Summary
- Fix spelling errors.
- Remove some trailing spaces.
- Replace unicode `’` with ASCII `'`.

### CODE REVIEW

**Comment by vitoyucepi** [comment]
I'm not sure if I should fix this mistake
https://github.com/savonet/ocaml-ssl/blob/dfba6793d939b46f55f0452bcbe04759286aa3fe/tests/ssl_certs.ml#L30

File: src/ssl.ml [hunk 1]
```diff
@@ -102,10 +102,10 @@ let () =
     | Method_error -> Some ("SSL: Method error")
     | Context_error -> Some ("SSL: Context error")
     | Certificate_error s -> Some ("SSL: Certificate error: " ^ s)
-    | Cipher_error -> Some ("SSL: Cihper error")
+    | Cipher_error -> Some ("SSL: Cipher error")
```
Comment (anmonteiro) [inline src/ssl.ml:105]: these were fixed in #109, please rebase

**Review by anmonteiro — APPROVED** [review]
thanks

File: src/ssl.ml (hunk 1 above)
Comment (vitoyucepi) [reply src/ssl.ml]: Here you are!

**Comment by anmonteiro** [comment]
Out of curiosity, did you run the repo through some automated typo-fixing tool?

**Comment by vitoyucepi** [comment]
I use spellchecker from JetBrains IDE.
It provides spelling + grammar suggestions.

**Pull request merged**